azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/appservice/AzureAppService.java

```diff
@@ -104,6 +108,11 @@ public IWebApp webapp(WebAppEntity webAppEntity) {
                 .collect(Collectors.toList());
     }
 
+    public @Nonnull List<Runtime> listWebAppRuntimes(@Nonnull OperatingSystem os, @Nonnull JavaVersion version) {
+        return Runtime.values().stream().filter(r -> Objects.equals(os, r.getOperatingSystem()) && Objects.equals(version, r.getJavaVersion()))
```

Comment:
`Runtime` includes runtime for web app and functions, we may need to add correspond constant/static method for each service

Reply:
this PR is only for webapp